fvm/fvm_bench_test.go

```diff
@@ -223,16 +261,49 @@ func (b *BasicBlockExecutor) SetupAccounts(tb testing.TB, privateKeys []flow.Acc
 	return accounts
 }
 
+type logExtractor struct {
+	InteractionUsed map[string]uint64
+}
+
+type txSuccessfulLog struct {
+	TXHash                string `json:"txHash"`
+	LedgerInteractionUsed uint64 `json:"ledgerInteractionUsed"`
+}
+
+func (l *logExtractor) Write(p []byte) (n int, err error) {
```

Comment:
nit: could use explanation of WHY we want to check for successful transactions, counting interactions, etc.